client/express-checkout/blocks/express-checkout-element.scss

```diff
@@ -14,14 +14,3 @@
 	margin: 24px 0 !important;
 	height: 20px;
 }
-
-// Checkout Block
-.wc-block-components-express-payment--checkout {
```

Comment:
Removing this looks fine with `trunk` now that the mininum with of the grid is [set to 240px](https://github.com/woocommerce/woocommerce/pull/49304) by WC core.  👍